cas-server-core-audit/src/main/java/org/apereo/cas/audit/spi/AssertionAsReturnValuePrincipalResolver.java

```diff
@@ -0,0 +1,55 @@
+package org.apereo.cas.audit.spi;
+
+import org.apereo.cas.authentication.Authentication;
+import org.apereo.cas.validation.Assertion;
+import org.apereo.inspektr.common.spi.PrincipalResolver;
+import org.aspectj.lang.JoinPoint;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Inspektr's principal resolver that first checks if the return value of the target joinpoint is instance
+ * of {@link org.jasig.cas.client.validation.Assertion} and then retrieves the principal from there.
+ *
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Empty line should be followed by <p> tag on the next line.](https://www.codacy.com/app/mmoayyed/cas/file/2639971502/issues/source?bid=3256115&fileBranchId=3321105#l13)